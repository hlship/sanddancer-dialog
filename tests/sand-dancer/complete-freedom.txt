%% start with one complete plan
(grab [#strength #blanket #luck #canned-oranges #grandmas-stories #oceans-perfume #graduation-night])
(grab [#heard-the-news #road-trips])
(setup sand-dancers offer)
u
i
say hello
ask who
say yes
ask freedom
i
trade job for freedom
trade last day for freedom
trade family guy for freedom
trade graduation for freedom
trade news for freedom
trade meeting ocean for freedom
trade road trips for freedom
trade perfume for freedom
trade stories for freedom
i
s
