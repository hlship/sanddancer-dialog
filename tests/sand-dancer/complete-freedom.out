

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> (grab [#strength #blanket #luck #canned-oranges #grandmas-stories #oceans-perfume #graduation-night])
Query succeeded: (grab [#strength #blanket #luck #canned-oranges #grandmas-stories #oceans-perfume #graduation-night])
> (grab [#heard-the-news #road-trips])
Query succeeded: (grab [#heard-the-news #road-trips])
> (setup sand-dancers offer)

Storage Room
Waves of sand flow like water through the room, filling it higher and higher.
You glance wildly at the exits south and up.

Query succeeded: (setup sand-dancers offer)
> u
Barely able to reach the ladder through sand up to your waist, you haul yourself
up the rungs and pull free just in time.

Roof
Except it's more like a foundation now, the building's sunk almost entirely into
the sand.

The sand swirls and churns like something liquid or alive around you, and then
you look and see it is alive, thousands and thousands of lizards, little sand-
dancers, are burrowing up from somewhere underground, scattering and writhing in
some complex pattern.

Then, suddenly, everything stops.

And with a mighty heave, a huge lizard crests through the sand right in front of
you.

The lizard's tongue snakes through the air, tasting it, tasting you. One slitted
eye studies you intently. It pauses, as if waiting for you to say something.

You could say hello.

> i
You have a flashlight, some canned oranges, an emergency blanket, your lighter,
your wallet, and your emotional baggage. You're wearing your denim jacket.
You've gained the talents strength and luck.

In your emotional baggage are the memories of your last day of high school,
watching Family Guy with Karl, meeting Ocean, your shit job, road trips through
the desert, when you heard the news, her graduation night, Ocean's perfume, and
your grandma's stories.

You could say hello.

> say hello
Stunned by what you see, you stammer out a greeting.

The ground rumbles as the lizards shift. "Hello," he says and his voice shakes
the earth and creaks the rusty joins of the tower. "Yes, hello, little one.
Strange thing to say when I've always been with you, watching, waiting. But
hello, hello at last, Nakaibito."

You could ask who he is or ask about the lizards.

> ask who
"Who, uh, what are you?" you stammer.

"I am Sand-dancer," he breathes flicking his toungue in and out, slowly. "I am
the desert and the dust and the rain and the wind. I was there the first time
you went hungry and the first time you made love and the last time you asked for
help, but I could not help you then. But tonight, Nakaibito, tonight I can."

"By your actions and through your talents, you have chosen to survive the night
here, alone" he says, his great slittled eye blinking. "You have looked into
yourself and found your story, the path you will walk tomorrow."

He breathes. "And are you happy with your story?" he asks, and there's some
dangerous undertone to the question and you don't know why. The lizards tense,
quivering, twisting their little heads back and forth as they await your answer.

You could say yes, say no, or say you're not sure.

> say yes
You pause for a moment and realize that every insane thing that's happened
tonight has led up to this instant. You open your mouth to speak, but find that
all you can manage is a little nod.

"Yes?" Sand-dancer says, and the lizards quiver anxiously. "You're happy with
the path you've chosen? A life on the run, alone, work time in one town, jail
time in another, no job, no skills, no family?" The lizards scurry angrily,
snapping and clawing at each other, with a sound like static and sand.

"It doesn't matter," he says more calmly, as the lizards subside, "since you
have already made your choice. You cannot take back your actions. You cannot
unmake the past." The lizards around him tilt their heads, edge forward
cautiosly, and he seems to grin. "But I can."

The little sand-dancers writher and chitter all around you.

A glistening egg rises from the sand, glowing with neon potential, the lizard
glances down at it almost dismissively. "This is spirit," he says, "the last
talent you need to achieve your goal, and you can have it for three memories.
But perhaps I have something else to trade."

The lizards whisper in anticipation as a second egg rises. This one's bigger,
and almost blinding with its intensity, and the lizards draw back, chittering in
awe and amazement. "This," Sand-dancer says, "is freedom. Pure, undiluted, and
absolute. Many crave it and few get a chance to take it. I offer it to you,
Nakaibito, but it does not come cheaply." He grins. "For freedom, you must give
up everything. You must give me all your memories."

The lizards begin to circle restlessly as they wait for you to choose. Sand-
dancer merely watches, and waits.

You could ask about spirit or ask about freedom.

> ask freedom
"Freedom, huh? I bet I could use some of that" you say.

"With freedom, Nakaibito, you can make your troubles vanish. True freedom is the
power of unmaking the past, of undoing what's done and regretted. This power is
great and terrible, but I promise you." He flicks his tongue. "It will set you
free."

You could ask about spirit.

> i
You have a flashlight, some canned oranges, an emergency blanket, your lighter,
your wallet, and your emotional baggage. You're wearing your denim jacket.
You've gained the talents strength and luck.

In your emotional baggage are the memories of your last day of high school,
watching Family Guy with Karl, meeting Ocean, your shit job, road trips through
the desert, when you heard the news, her graduation night, Ocean's perfume, and
your grandma's stories.

You could ask about spirit.

> trade job for freedom
He flicks out his tongue and pulls the memory of your shit job from your core,
swallowing it whole. "A good start," he says, "but you must trade more."

You could ask about spirit.

> trade last day for freedom
He flicks out his tongue and pulls the memory of your last day of high school
from your gut, swallowing it whole. "Yes," he says, "but more."

You could ask about spirit.

> trade family guy for freedom
He flicks out his tongue and pulls the memory of watching Family Guy with Karl
from your head, swallowing it whole. "More," he says.

You could ask about spirit.

> trade graduation for freedom
He flicks out his tongue and pulls the memory of her graduation night from your
core, swallowing it whole. "Still more," he says.

You could ask about spirit.

> trade news for freedom
He flicks out his tongue and pulls the memory of when you heard the news from
your soul, swallowing it whole. "I must have more," he says.

You could ask about spirit.

> trade meeting ocean for freedom
He flicks out his tongue and pulls the memory of meeting Ocean from your heart,
swallowing it whole. "Give me more," he says.

You could ask about spirit.

> trade road trips for freedom
He flicks out his tongue and pulls the memory of road trips through the desert
from your gut, swallowing it whole. "Give me more," he says.

You could ask about spirit.

> trade perfume for freedom
He flicks out his tongue and pulls the memory of Ocean's perfume from your head,
swallowing it whole. "Give me more," he says.

You could ask about spirit.

> trade stories for freedom
Sand-dancer nods. "Yes," he says, "a fair trade." And something happens inside
you as he says it. The memory of your grandma's stories shifts and wriggles and
fades away and it's still there, but now it's shifted, become something else.
And, yeah, it kinda does feel like you could call it freedom. Cool. "The choice
is made," he rumbles, "your path is set. Goodbye, Nakaibito." And he turns and
dives into the sand. The lizards twist and writhe and follow him.

And then, just as quiet settles, another huge rumble shakes the ground, and,
almost like an afterthought, the building breaches the sand and rises to the
surface.

Crumbling Concrete
This concrete building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned. Tendrils of desert sand drift
against a boarded-up door to the northeast leading inside; you could also walk
around to the east or head south back to your truck.

> i
You have a flashlight, some canned oranges, an emergency blanket, your lighter,
your wallet, and your emotional baggage. You're wearing your denim jacket.
You've gained the talents strength, luck, and freedom.

> s
You go, with a strange, kinda tingly feeling in the backs of your hands and
inside your eyes. You feel powerful, stronger than the wind now and free, most
of all free. You walk backwards across the sand to your truck, sucking up your
footprints as you go, and slip behind the wheel. There's this crazy wrenching
bang and you jerk backwards as the engine starts, uncrashing your pickup off the
cactus and juddering back onto the highway, and you smile. You're going back,
back to a second chance, back to make it all right.

You speed up, drive backwards at a thousand miles an hour to work, and unfix
junkers and repack parts in a flash, faster and faster. The days rewind like
videotape, unwork, unsleep, giving back kisses from Ocean one by one. And then
it's that night, the night you made the Big Mistake before your luck started
turning around, yeah, cause this time you unmake love. And the moment unpasses
and your future is safe and uncomplicated, no kid, no nightmares, no arguments,
no lectures, and you just have to get this thing out of reverse now and do it
right this time (or mebbe not at all, it's too hot anyway ya know babe?) and
this time you'll make everything perfecto.

But you don't turn around. You try, but it's kinda like trying to stop falling,
and you suddenly get that you are falling, plummeting back faster and faster
like you got pushed off the cliff of now. You're unmeeting Ocean and unditching
school and unsmoking for the first time behind the dumpsters, faster and faster,
stomach in your throat, memories blowing back your hair as they rocket past.
You're a kid again unskateboarding and unlearning video game combos, screaming
backwards faster and faster and you get it now, you get the lizard's joke. He's
making you free by unmaking you, fixing the burden by erasing the guy who's
bearing it, and you're unlearning to read and unlearning to walk and uncrawling
and uncrying and then some brilliant moment of light and noise and chaos comes
shrieking towards you hella fast, fast, faster than anything and it's too late,
you're crashing, you're crashing, you crash...

     *** You have freedom. ***

Would you like to:
     UNDO the last move,
     RESTORE a saved position,
     QUIT the program,
     or RESTART from the beginning?
> 
